pyang/translators/yang.py

```diff
@@ -24,18 +24,36 @@ def add_opts(self, optparser):
             optparse.make_option("--yang-remove-unused-imports",
                                  dest="yang_remove_unused_imports",
                                  action="store_true"),
+            optparse.make_option("--yang-newline-policy",
+                                 dest="yang_newline_policy",
+                                 choices=newline_policies,
+                                 default="pyang",
+                                 help="Newline policy POLICY.  "
+                                      "Default 'pyang'. Supported "
+                                      "policies are: " + ', '.join(
+                                         list(newline_policies))),
             ]
         g = optparser.add_option_group("YANG output specific options")
         g.add_options(optlist)
 
+    def setup_fmt(self, ctx):
+        ctx.implicit_errors = False
```

Comment:
Note that I added this because I felt that there was no value in getting warnings from imported modules. I note that most/all of the output formats do the same... so I am a bit worried that I've missed something here, and that that there might be a good reason _not_ to do this for the YANG output format.